src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+log4j.rootLogger=INFO, stdout, file
+log4j.logger.org.hibernate=INFO
+log4j.logger.org.hibernate.SQL=DEBUG
+log4j.logger.org.hibernate.type.descriptor.sql=INFO
```

Comment:
do you want to save hibernate logs intentionally?

Reply:
Yes. But I can remove this)

Reply:
it is ok if the logs remain clear